src/interviewer.t

```diff
@@ -217,17 +235,70 @@ warner_interviewer : Person 'interviewer' 'Harried Interviewer' @interview_room
 ;
 
 ++ SpecialTopic 'anime club' ['anime', 'anime club', 'japanese', 'japanese club', 'weeaboo', 'weeb']
-    ""
+    ""//"<q>私はアニメが大好きです~~~!</q>"
```

Comment:
Even running it in web mode through frob, it's unrecognized. These are UTF-8, right?
I'll merge now and fix a bug I just found in the build file. I also might move all people files to their own dir, so be ready to pull again before you develop.